From ddd56831b444501c06898c763a836d1a18ee85a1 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 6 Mar 2018 09:42:15 -0600
Subject: [PATCH 04/25] vendor: fix ro.adb.secure for vendor-building devices

* This ensures the prop is set correctly for devices
  that build a vendor image by placing the prop in
  /system/etc/prop.default, rather than /vendor/default.prop
  so it'll be in a prop file that is run for mangle_default_prop
  in build/make/tools/post_process_props.py, therby
  setting persist.sys.usb.config correctly to "none".

Change-Id: I4027541cf1eae9dd967636efe35de9578922b725
---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 502d915a..caa10406 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -28,7 +28,7 @@ endif
 
 ifneq ($(TARGET_BUILD_VARIANT),eng)
 # Enable ADB authentication
-PRODUCT_DEFAULT_PROPERTY_OVERRIDES += ro.adb.secure=1
+PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 endif
 
 ifeq ($(BOARD_CACHEIMAGE_FILE_SYSTEM_TYPE),)
-- 
2.17.1

